xclim/data/fr.json

```
@@ -141,330 +141,283 @@
   },
   "TG": {
     "long_name": "température journalière moyenne",
     "description": "température journalière moyenne estimée par la médiane des températures maximales et minimales.",
     "title": "Moyenne des températures maximales et minimales",
-    "comment": "",
     "abstract": "La température journalière moyenne en assumant une distribution symétrique des températures.  Tg = (Tx + Tn) / 2"
   },
   "TG_MIN": {
     "long_name": "Minimum de la température journalière moyenne",
     "description": "Minimum {freq:m} de la température journalière moyenne.",
     "title": "Minimum de la température journalière moyenne",
-    "comment": "",
     "abstract": "Minimum de la température journalière moyenne."
   },
   "TG_MAX": {
     "long_name": "Maximum de la température journalière moyenne",
     "description": "Maximum {freq:m} de la température journalière moyenne.",
     "title": "Maximum de la température journalière moyenne",
-    "comment": "",
     "abstract": "Maximum de la température journalière moyenne."
   },
   "TG_MEAN": {
-    "long_name": "Moyenne de la température journalière moyenne",
-    "description": "Moyenne {freq:f} de la température journalière moyenne.",
-    "title": "Moyenne de la température journalière moyenne",
-    "comment": "",
-    "abstract": "Moyenne de la température journalière moyenne."
+    "long_name": "Moyenne de la température journalière",
+    "description": "Moyenne {freq:f} de la température journalière.",
+    "title": "Température journalière moyenne",
+    "abstract": "Température journalière moyenne."
   },
   "TG10P": {
     "long_name": "Nombre de jours où la température est au-dessus du 10e percentile",
     "description": "Nombre {freq:m} de jours où la température est au-dessus du 10e percentile. Le 10e percentile est calculé à partir d'une fenêtre de 5 jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence.",
     "title": "Nombre de jours où la température est au-dessus du 10e percentile",
-    "comment": "",
     "abstract": "Nombre de jours où la température est au-dessus du 10e percentile"
   },
   "TG90P": {
     "long_name": "Nombre de jours où la température est au-dessus du 90e percentile",
     "description": "Nombre {freq:m} de jours où la température est au-dessus du 90e percentile. Le 90e percentile est calculé à partir d'une fenêtre de 5 jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence.",
     "title": "Nombre de jours où la température est au-dessus du 90e percentile",
-    "comment": "",
-    "abstract": "Number of days with daily mean temperature over the 90th percentile."
+    "abstract": "Nombre de jours où la température est au-dessus du 90e percentile."
   },
   "TN_MIN": {
-    "long_name": "Minimum de la température journalière minimale",
+    "long_name": "Minimum de la température journalière",
```

I would leave it as it were, since this new name loses the notion that Tmin specifically was used.

---

@EmilieBresson je te lance dans la mêlée.

---

@RondeauG  C'était pour éviter la répétition qui n'a pas de sens puisque le minimum du minimum c'est le minimum... (même principe avec la moyenne de la moyenne). Si ça ne passe pas au conseil OK.

---

Mais dans cette version-ci, à part pour le champs "description", il n'y a rien qui distingue TX_MIN (min du max), TG_MIN (min de la moyenne) et TN_MIN (min du min), alors que ce sont trois indicateurs différents ?
Ça peut sembler répétitif, mais je crois que c'est un endroit où on peut se le permettre. Mais en même temps, ce n'est pas critique non plus.

---

Au contraire, non? On aurait : « Minimum de la température journalière maximale », « Minimum de la température journalière moyenne » et « Minimum de la température journalière ».

---

On s'en jase en janvier avec des avis extérieurs ?

---

@aulemahal Mais c'est justement ici ce qui est voué à être retiré ici, non?
"Minimum de la température journalière minimale" --> "Minimum de la température journalière"

---

Oui, mais le "minimal" ne serait retiré _que_ pour `tn_min` puisque c'est le seul endroit où il est redondant?  `tx_min` et `tg_min` ne sont pas touchés.

---

Ouais... J'ai peur que ça puisse quand même porter à confusion, mais OK.

---

En fait est-ce que : 
```suggestion
    "long_name": "Minimum de la température",
```
serait encore moins confus? Tsé, le "journalier" n'a plus rien à voir à ce stade.

---

Mauvaise idée à mon avis. Parce que ça pourrait être de la température horaire initialement et tu pourrais avoir des résultats différents.
Faut discuter de ça avec un plus gros échantillon d'utilisateurs...